src/main/java/uk/gov/hmcts/reform/jps/controllers/RecordSittingRecordsController.java

```diff
@@ -40,6 +57,16 @@ public class RecordSittingRecordsController {
     private final SittingRecordService sittingRecordService;
     private final LocationService regionService;
 
+
+    @Operation(description = "Create a new sitting record")
+    @ApiResponse(responseCode = "201",
+        content = @Content(schema = @Schema(implementation = RecordSittingRecordResponse.class)),
+        description = "Successfully created sitting record")
+    @ApiResponse(responseCode = "200", description = RESPONSE_200)
+    @ApiResponse(responseCode = "400", description = RESPONSE_400, content = @Content)
+    @ApiResponse(responseCode = "401", description = RESPONSE_401, content = @Content)
+    @ApiResponse(responseCode = "403", description = RESPONSE_403, content = @Content)
+
     @PostMapping(
         path = {"", "/{hmctsServiceCode}"}
```

Comment:
I think the blank quotes might cause two endpoints in swagger one for POST /recordSittingRecords and one for POST /recordSittingRecords/{hmctsServiceCode}. Can we double check?

Reply:
Yes it does so removed blank quotes and missing path parameter endpoints

Reply:
![image](https://github.com/hmcts/jps-judicial-payment-service/assets/69344273/c84751e7-f5b6-4378-a3ef-1235d25fe59a)